usage/rest-client/src/main/java/brooklyn/rest/client/BrooklynApi.java

```diff
@@ -197,6 +216,11 @@ public AccessApi getAccessApi() {
     }
 
     public static <T> T getEntity(Response response, Class<T> type) {
```

Comment:
It looks like this method isn't called (or is it reflective)?

If it's not useful, but are concerned about API compatibility, then could mark as deprecated and delete in 0.8.0-SNAPSHOT?